exchange/exchangetest/aliases.json

```diff
@@ -64,16 +64,7 @@
                 }
               }
             }
-          ],
```

Comment:
Simplifying tests by not hardcoding in a server object to the test exchange?

Reply:
Exactly. The server object tests are now isolated to their own cases. Values default to empty and can be specified in the test case json file for those which need it. IMHO the number of tests affected by changes to this code is too much.